### PR TITLE
Add pager for comments on group overview

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/30400

screenshot_20180123_123430

Still needs to be tested.

### CODE REVIEW

File: lib/OpenQA/WebAPI/Controller/Main.pm [hunk 1]
```diff
@@ -97,15 +97,37 @@ sub group_overview {
...
+        # find comments
+        my $page               = int($self->param('comments_page') // 1) // 1;
+        my $comments_resultset = $self->app->schema->resultset('Comments');
+        my $comments           = $comments_resultset->search(
+            {
+                group_id => $group->id
+            },
+            {
+                page     => $page,
+                rows     => 5,
```
Comment (coolo) [inline lib/OpenQA/WebAPI/Controller/Main.pm]: the 5 is a big magic - and repeated here and in the template. Better pass it as param from the template and use a variable there.

Comment (Martchus) [reply lib/OpenQA/WebAPI/Controller/Main.pm]: True. I have to change that code anyways because having some "..." in the gap would be nice.

**Comment by Martchus** [comment]
I fixed the magic 5 and added … between the gaps like the data tables do. The template code is still a bit messy, but it should work now. There's also a basic test.

Still we should figure out why the markdown rendering is so horribly slow.

**Comment by okurz** [comment]
Awesome, thanks a lot

**Pull request merged**